docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,37 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8 - September 4, 2024
+
+### New Features
+
+- Introduced the notion of Migration complexity in assessment and analyze-schema reports, which range from LOW to MEDIUM to HIGH. For PostgreSQL source, this depends on the number and complexity of the PostgreSQL features present in the schema that are unsupported in YugabyteDB.
+- Introduced a bulk assessment command (`assess-migration-bulk`) for Oracle which allows you to assess multiple schemas in one or more database instances simultaneously.
+- Streamlined the airgapped installation process. This includes clearer instructions for installing dependencies; providing a script to verify dependencies are installed; and providing a bundle that includes all the Voyager packages and scripts.
+- Support fall-forward/fall-back live migration workflows using the PostgreSQL replication protocol-based CDC connector.
+
+### Enhancements
+
+- Enhanced assessment and analyze reports to detect, and specify workarounds for PostgreSQL features that are incompatible with YugabyteDB. For example, Multi-column GIN indexes, GENERATED ALWAYS AS STORED columns, ALTER TABLE variants, Deferrable constraints, ALTER TABLE ADD PRIMARY KEY on a partitioned table, and so on.
+- Enhanced Manual Review Guidelines section for PostgreSQL source to list known issues that are unsupported by YugabyteDB, with workarounds.
```

Comment:
Link the `Manual Review Guidelines` to this https://deploy-preview-23764--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/known-issues/postgresql/